org.sil.FieldWorks.yml

```diff
@@ -73,8 +75,12 @@ finish-args:
   # Shared writing system and encoding converters data location used by FieldWorks Debian packages, that may have
   # user-created data. Be able to read from this location in order to migrate from it.
   - --filesystem=/var/lib/fieldworks:ro
+  - --filesystem=xdg-run/dconf
+  - --filesystem=xdg-config/dconf:ro
+  - --env=GTK_IM_MODULE_FILE=/app/lib/gtk-2.0/2.10.0/immodules.cache
 
 modules:
+  - shared-modules/gtk2/gtk2.json
```

Comment:
Let's make this a dependency of the fieldworks module, rather than a parallel item. Unless I'm not thinking about this the right way?

Reply:
Actually, maybe this should go in place of where `-name: gtk+2.0` was. This gtk2.json will be satisfying the need that that was doing forg gtkmm2.4.